api/v1alpha3/baremetalcluster_types.go

```diff
@@ -90,6 +90,7 @@ type BareMetalClusterStatus struct {
 // +kubebuilder:printcolumn:name="Ready",type="string",JSONPath=".status.ready",description="BaremetalCluster is Ready"
 // +kubebuilder:printcolumn:name="Error",type="string",JSONPath=".status.failureReason",description="Most recent error"
 // +kubebuilder:printcolumn:name="Cluster",type="string",JSONPath=".metadata.labels.cluster\\.x-k8s\\.io/cluster-name",description="Cluster to which this BMCluster belongs"
+// +kubebuilder:printcolumn:name="APIEndpoints",type="string",JSONPath=".spec.controlPlaneEndpoint",description="API endpoint"
```

Comment:
The name is plural while the JSONPath='...' refers to a single value. 
Is there a mismatch ?

Reply:
done